### ISSUE TITLE
Mask_loss is increasing.

### ISSUE THREAD

**Issue by Arham-Aalam** [matterport/Mask_RCNN#2001]
I want to segment sports ground(field) images, I have 1000 ground images and I have attached images of all losses below, The issue is all validation losses are decreasing well except Mask Loss. any suggestion why it is not decreasing?
Configurations:
BACKBONE                       resnet50
BACKBONE_STRIDES               [4, 8, 16, 32, 64]
BATCH_SIZE                     1
BBOX_STD_DEV                   [0.1 0.1 0.2 0.2]
COMPUTE_BACKBONE_SHAPE         None
DETECTION_MAX_INSTANCES        10
DETECTION_MIN_CONFIDENCE       0.7
DETECTION_NMS_THRESHOLD        0.3
FPN_CLASSIF_FC_LAYERS_SIZE     1024
GPU_COUNT                      1
GRADIENT_CLIP_NORM             5.0
IMAGES_PER_GPU                 1
IMAGE_CHANNEL_COUNT            3
IMAGE_MAX_DIM                  1024
IMAGE_META_SIZE                14
IMAGE_MIN_DIM                  800
IMAGE_MIN_SCALE                0
IMAGE_RESIZE_MODE              square
IMAGE_SHAPE                    [1024 1024    3]
LEARNING_MOMENTUM              0.9
LEARNING_RATE                  0.001
LOSS_WEIGHTS                   {'rpn_class_loss': 1.0, 'rpn_bbox_loss': 1.0, 'mrcnn_class_loss': 1.0, 'mrcnn_bbox_loss': 1.0, 'mrcnn_mask_loss': 1.0}
MASK_POOL_SIZE                 14
MASK_SHAPE                     [56, 56]
MAX_GT_INSTANCES               100
MEAN_PIXEL                     [123.7 116.8 103.9]
MINI_MASK_SHAPE                (56, 56)
NAME                           ground
NUM_CLASSES                    2
POOL_SIZE                      7
POST_NMS_ROIS_INFERENCE        1000
POST_NMS_ROIS_TRAINING         2000
PRE_NMS_LIMIT                  6000
ROI_POSITIVE_RATIO             0.33
RPN_ANCHOR_RATIOS              [0.5, 1, 2]
RPN_ANCHOR_SCALES              (32, 64, 128, 256, 512)
RPN_ANCHOR_STRIDE              1
RPN_BBOX_STD_DEV               [0.1 0.1 0.2 0.2]
RPN_NMS_THRESHOLD              0.7
RPN_TRAIN_ANCHORS_PER_IMAGE    128
STEPS_PER_EPOCH                200
TOP_DOWN_PYRAMID_SIZE          256
TRAIN_BN                       False
TRAIN_ROIS_PER_IMAGE           100
USE_MINI_MASK                  True
USE_RPN_ROIS                   True
VALIDATION_STEPS               50
WEIGHT_DECAY                   0.0001

![Screenshot from 2020-02-15 12-34-39](https://user-images.githubusercontent.com/23229365/74583889-246ca000-4ff2-11ea-9415-6ea6cd23e5a6.png)
![Screenshot from 2020-02-15 12-34-58](https://user-images.githubusercontent.com/23229365/74583892-2a628100-4ff2-11ea-8e29-26d6a58c577a.png)
![Screenshot from 2020-02-15 12-35-12](https://user-images.githubusercontent.com/23229365/74583894-2df60800-4ff2-11ea-8bcb-e01e1a4e01c2.png)
![Screenshot from 2020-02-15 12-32-48](https://user-images.githubusercontent.com/23229365/74583896-30f0f880-4ff2-11ea-9c3f-ed3ed5a762b9.png)

**Comment by suchiz** [comment]
Have you checked if your masks were well loaded in load_mask ?

**Comment by Arham-Aalam** [comment]
Yes they are loading and I'm getting the Mask accuracy around 50% but the model is unable to cover boundary region, every time it is predicting middle part of the object.

**Comment by purugupta99** [comment]
Did you find any fix for this? I'm facing similar issues with the Mask Branch.

**Comment by Arham-Aalam** [comment]
No luck yet!!
